elastalert/loaders.py

```diff
@@ -71,8 +71,6 @@ def load_rule_schema():
 
 
 class RulesLoader(object):
-    # import rule dependency
-    import_rules = {}
```

Comment:
Moved into `__init__` to stop test cases from bleeding together.

Reply:
See https://github.com/jertel/elastalert2/runs/4360268026?check_suite_focus=true

> =================================== FAILURES ===================================
> _______________________ test_load_yaml_recursive_import ________________________
> [gw1] linux -- Python 3.10.0 /home/elastalert/tests/.tox/py310/bin/python
> 
>     def test_load_yaml_recursive_import():
>         config = {}
>         rules_loader = FileRulesLoader(config)
>     
>         root_path = os.path.join(loaders_test_cases_path, 'recursive_import/root.yaml')
>         branch_path = os.path.join(loaders_test_cases_path, 'recursive_import/branch.yaml')
>         leaf_path = os.path.join(loaders_test_cases_path, 'recursive_import/leaf.yaml')
>     
>         leaf_yaml = rules_loader.load_yaml(leaf_path)
>         assert leaf_yaml == {
>             'name': 'leaf',
>             'rule_file': leaf_path,
>         }
> >       assert sorted(rules_loader.import_rules.keys()) == [
>             branch_path,
>             leaf_path,
>         ]
> E       AssertionError: assert ['/home/elast..., 'blah.yaml'] == ['/home/elast...rt/leaf.yaml']
> E         Left contains one more item: 'blah.yaml'
> E         Use -v to get the full diff

Reply:
Demo of static vs member:

```
class TestObject(object):
    static_list = []
    def __init__(self):
        self.member_list = []

obj1 = TestObject()
obj1.static_list.append('static1')
obj1.member_list.append('member1')

obj2 = TestObject()
obj2.static_list.append('static2')
obj2.member_list.append('member2')

print('static_list:')
print(obj2.static_list)

print('member_list:')
print(obj2.member_list)
```

Outputs:

```
static_list:
['static1', 'static2']
member_list:
['member2']
```

Reply:
Rules loader only gets created once:

https://github.com/jertel/elastalert2/blob/878635c2c8dbb2975e2d893142145b390dfd74ee/elastalert/config.py#L101

So I believe the static variable was just an oversight.